universal-login-react/src/core/utils/toTopUpComponentType.ts

```diff
@@ -0,0 +1,13 @@
+import {TopUpProvider} from '../models/TopUpProvider';
+import {TopUpComponentType} from '../models/TopUpComponentType';
+
+export const toTopUpComponentType = (provider: TopUpProvider): TopUpComponentType => {
+  switch (provider) {
+    case TopUpProvider.RAMP:
+      return TopUpComponentType.ramp;
+    case TopUpProvider.SAFELLO:
+      return TopUpComponentType.safello;
+    case TopUpProvider.WYRE:
+      return TopUpComponentType.wyre;
+  }
```

Comment:
That feels a bit redundant. 
TopUpComponentType and TopUpProvider seem like the same thing.
I would merge them.